### DISCUSSION TITLE
How to publish additional information

### DISCUSSION THREAD

**Discussion by avhoboken** [dawidchyrzynski/arduino-home-assistant#107] [General]
I'm looking for the possibilty to publish simple diagnostic information about a device, information like for instance the IP address, I see that the HAMqtt class has several functions like publish() and writepayload(), are these meant for that?
Or is there another (better) way to do that?

Thanks.

**Comment by dawidchyrzynski** [comment]
Hi @avhoboken,

This topic is already described in the documentation: https://dawidchyrzynski.github.io/arduino-home-assistant/documents/library/mqtt-advanced.html
